### ISSUE TITLE
Favorites Quickaccess Sorting 2.0 does not work with Nextcloud 26.0.0

### ISSUE THREAD

**Issue by 42ske** [newhinton/quickaccesssorting#13]
After intalling Favorites Quickaccess Sorting 2.0 on Nextcloud 26.0.0 there are no favorties sorting-settings within the files app nor is it possible to sort the favorites via drag-n-drop.

**Comment by newhinton** [comment]
Yeah, i did not expect an update to the files app. I have to rebuild the javascript fully, because quite a lot that i depend on changed. 

But i have already started working on the update, albeit it might take quite a while.

**Comment by newhinton** [comment]
So i do have a working prototype now.

I have "published" [v3.0.0](https://github.com/newhinton/quickaccesssorting/releases/tag/v3.0.0), but not to the appstore. That version re-implements all the required features, but there is quite a few visual bugs that i'd like to adress first.

**Comment by 42ske** [comment]
Hi @newhinton sorry that I didn’t test it before. I just updated and activated version 3.0. It is shown in the files app preferences but it is not working on my installation (NC 26.0.1): I cannot select the order-method. The field stays empty… Please let me know if you need further information.

**Comment by newhinton** [comment]
There is currently no visual feedback. Can you check if the sidebar sorts as you would expect from your selection?

**Comment by 42ske** [comment]
I cannot see any effect with two date sorted and the alphabetical sorted mode. With the custom orderded mode it is the foloowing: after a reload I can reorder the favorites. After a second reload the order is changed in a random way.

**Comment by newhinton** [comment]
It wont be in a random way. Do you know how to use the developer console and are willing to check a few things?

**Comment by 42ske** [comment]
Since now, I don’t but if you send me a link I will have a look and try to check it out in the next days. Today I sadly have to do some other urgent things.

**Comment by newhinton** [comment]
You can press F12 to open (and close) the developer console on firefox or chrome.

Then you can click on the network-tab inside the developer console. It might be hidden behind the two arrows ">>", as in the following screenshot:
![grafik](https://github.com/newhinton/quickaccesssorting/assets/25279821/21c155d1-d695-4171-98e3-3fd45773eced)


There will be a "filter" input in the second row, there you can enter "quickaccess" to only get fitting results.

Now, if you reload the actual page of nextcloud, that list should populate.

There should be at least two results:
One contains "SortingStrategy" and one contains "CustomSortingOrder"
(except when you choose alphabetical, then its only the first.)


If you now click on that entry, it should look roughly like this:

![grafik](https://github.com/newhinton/quickaccesssorting/assets/25279821/007832b3-b961-4753-afd5-64df34312f07)
The "result" tab-content is what is relevant here.
